e2e/mycluster.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: moco.cybozu.com/v1beta1
```

Comment:
Do we need this file?

Reply:
I certainly may not need it.
I will remove it.

Reply:
Sorry, I committed another unnecessary file.
I remove it.

Reply:
I deleted the file and squashed the commit.
https://github.com/cybozu-go/moco/pull/471/commits/4548faf31bb4d5341048be6d40add60478a264ce